frontend/trpc/routes/documents/internal.ts

```
@@ -0,0 +1,124 @@
+      if (!document) throw new TRPCError({ code: "INTERNAL_SERVER_ERROR", message: "Failed to create document" });
+
+      return {
+        documentId: document.id,
+        fileStorageKey: document.fileStorageKey,
+        status: document.status
+      };
+    }),
+
+  getSigningData: companyProcedure
+    .input(z.object({ documentId: z.bigint() }))
+    .query(async ({ ctx, input }) => {
```

ESLint

[error] Replace `⏎····.input(z.object({·documentId:·z.bigint()·}))⏎····` with `.input(z.object({·documentId:·z.bigint()·}))`

(prettier/prettier)